force-app/main/default/lwc/lwcChatbotQuery/lwcChatbotQuery.js

```diff
@@ -73,12 +80,19 @@ export default class LwcChatbotQuery extends LightningElement {
         this.initAndQuery(false);
         return;
       }
-      // Store the lastQueryUid, this is useful to send analytics events.
-      this.lastQueryUid = searchAPIResponse.searchUid;
-      // Send a search event. This is async but no need to wait for it 
-      // since we don't want to block the execution and display of results.
-      this.sendSearchEvent(searchAPIResponse);
-      this.results = searchAPIResponse.results;
+      this.isLoading = false;
+      if (searchAPIResponse.status === 200) {

```

Comment:
`searchAPIResponse.status` is undefined when response is **ok**

Reply:
Well I just checked and didn't see a status, it's the body of the response btw not the "status" of the http request.